packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionHistoryTooltip.tsx

```diff
@@ -1,31 +0,0 @@
-import Tippy from '@tippyjs/react'
-import { BellIcon, XIcon } from '@heroicons/react/outline'
-
-export const TransactionHistoryTooltip = ({
-  isVisible,
-  onClose,
-  children
-}: {
-  isVisible: boolean
-  onClose: () => void
-  children: React.ReactElement
-}) => {
-  return (
-    <Tippy
-      visible={isVisible}
-      theme="light"
-      interactive
-      content={
-        <div className="flex items-center gap-1 text-dark">
```

Comment:
do we want to move the tooltip styles into a component?

Reply:
not sure if we will utilize it that often, and personally not a big fan of keeping unused components

Reply:
we can move it to component lib

Reply:
I think we should implement all the variations of tooltips provided in the UI in the component library itself, or probably have another feature ticket for it. Will need to cover all the 3-4 tooltip cases there.

Reply:
i'm in favour of just-in-time development for the component lib
happy to see this component moved to the repo with only 1 case for now